mypy/server/update.py

```diff
@@ -282,7 +282,7 @@ def update_single(self, module: str, path: str) -> Tuple[List[str],
         propagate_changes_using_dependencies(manager, graph, self.deps, triggered,
                                              {module},
                                              self.previous_targets_with_errors,
-                                             graph)
+                                             self.manager.modules)
```

Comment:
`propagate_changes_using_dependencies` also gets `manager` and `graph` as arguments, so it looks like the final argument is not actually required at all.